kolibri/auth/migrations/0003_auto_20170621_0958.py

```diff
@@ -6,14 +6,21 @@
 
 import kolibri.core.fields
 import kolibri.utils.time
-from django.db import migrations
+from django.db import OperationalError, migrations
 
 
 def convert_datetime_to_datetimetz(apps, schema_editor, model_name=None):
     if model_name:
         Model = apps.get_model("kolibriauth", model_name)
-        for model in Model.objects.all():
-            model.save()
+        try:
+            # Prevent the non-existence of this table from blowing up test runs
+            # Seems to only occur on test runs on Travis, but otherwise works fine
+            # Hopefully future migration squashes should prevent this code being
+            # called at all for new users
+            for model in Model.objects.all():
+                model.save()
+        except OperationalError:
+            pass
```

Comment:
Maybe add a warning here, so we know what's happening?

Reply:
Can do - although I added this exception because of a weird error that happened *only* on Travis, could not replicate locally.